### PR TITLE
5.0.7/R10 release notes

### PR DESCRIPTION
**Pull request by lnewson**
Here's my initial draft of the release notes for the R10 release. I've also made some edits/reworks/typo fixes to the premium skins page. As always, please feel free to make any changes directly.

@mhideo I know you asked to review the "Known issues" section for this, since we had the one premium skins issue with the preview plugin, so feel free to fix it and anything else up as you see fit :smiley: 

Note: I had to update the api build script to work with our new monorepo dev package.

### CODE REVIEW

**Comment by lnewson** [comment]
I should note that the Tiny Drive 1.3 docs [PR](https://github.com/tinymce/tinymce-docs/pull/1061) hasn't been merged yet, so if you load it locally or use launcher, then the links to the Tiny Drive changelog will still be 1.2.1.

**Comment by mhideo** [comment]
Thanks Lee. I've reviewed the known issues content. Looking forward to the next release :-)

File: release-notes/release-notes507.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+#### Help API
+
+The **Help** plugin includes a new `help_tabs` setting and an API to be able to add tabs from within custom plugins. This allows for the default tabs to be enabled/disabled, change the tab order or to add additional tabs to the help dialog.
+
+For more information on the new `help_tabs` setting and API refer to the [documentation]({{site.baseurl}}/plugins/help/).
+
+#### Image Tools custom image fetch
+
+The **Image Tools** plugin includes a new `imagetools_fetch_image` setting, which allows a custom function to be used to fetch the image content for more complex, such as when authentication is required.
```
Comment (EkimChau) [inline release-notes/release-notes507.md]: the image content for more complex, such as when authentication is required.

the image content in complexed environments where extra customisation is required such as authentication

Comment (lnewson) [reply release-notes/release-notes507.md]: Updated, though I did some commas and changed `complexed` to `complex`.

Comment (lnewson) [reply release-notes/release-notes507.md]: and oops, it was meant to say "complex situations" originally sorry!

**Pull request merged**